test/TestProjectWithIncludedSyntaxErrors/Post-Deployment/DataScript.sql

```diff
@@ -0,0 +1,3 @@
+print 'Loading file '
```

Comment:
Do we need all these test files just to trigger the error? Feels like a contrived example scenario that could be simplified for easier maintainability.